third_party/amd/backend/compiler.py

```diff
@@ -157,6 +157,7 @@ def make_llir(src, metadata, options):
         passes.convert.add_cf_to_llvmir(pm)
         passes.convert.add_arith_to_llvmir(pm)
         passes.common.add_canonicalizer(pm)
+        amd.passes.ttgpuir.add_extern_to_llvmir(pm)
```

Comment:
change pass name ?

Reply:
`lower_builtin_func_to_llvmir`? @antiagainst probably will have a better name idea :)

Could you add a comment explaining why we need that?

Reply:
I'll keep modifying the pass name till the end, after I finish fixing the lit tests, so that people get a chance to suggest names!